AppAuth-JS/built/token_response.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork